frontend-project/src/global.tsx

```diff
@@ -15,8 +15,8 @@ if (pwa) {
     );
   }); // Pop up a prompt on the page asking the user if they want to use the latest version
 
-  window.addEventListener('sw.updated', event => {
-    const e = event;
+  window.addEventListener('sw.updated', (event: Event) => {
+    const e = event as CustomEvent;
```

Comment:
Czy ja dobrze rozumiem, że `Event` i `CustomEvent` są przypięte do global scope?

Reply:
tak, o ile w tsconfig importowana jest biblioteka `DOM`.
Z tego co zauważyłem interfejsowi Event brakuje metod, które powinny się tam pojawić, za to CustomEvent używa generyków i sprawia mniej problemów